vsphere/assets/configuration/spec.yaml

```diff
@@ -292,6 +292,36 @@ files:
           display_default: ['vm', 'host']
           items:
             type: string
+      - name: include_events
+        description: |
+          The events that should be collected by the vSphere integration.
+
+          Each event is an object represented with the event name itself as a key.
+          However, while the keys should be events to include, the value should be an
+          array of message regexes to exclude/filter out. For example, if the key
+          is 'AlarmStatusChangedEvent' and the values for this key are 'Gray to Green' and 'Green to Gray',
+          then all events name 'AlarmStatusChangedEvent' that don't have the option of 'Gray to Green'
+          and 'Green to Gray' will be included.
+          
+          Currently, events can only be collected from the following resources:
+          VMs, hosts, datacenters, datastores, and clusters.
+
```

Comment:
Can we let customers know what the default behavior is here or link to the readme https://github.com/DataDog/integrations-core/tree/master/vsphere#events?

Reply:
Ah could we also add a note in the readme to say that these are the default events and that more can be added using `include_events` (linking to this option)?